documentation/source/install.rst

```diff
@@ -107,6 +107,10 @@ Installation of cocotb
 
           pip install cocotb
 
+.. note:: If your user does not have permissions to install cocotb using the instructions above,
+          try adding the ``--user`` option to ``pip``
+          (see :ref:`the pip documentation<https://pip.pypa.io/en/stable/user_guide/#user-installs>`).
```

Comment:
rST nit:

```suggestion
.. note::

    If your user does not have permissions to install cocotb using the instructions above,
    try adding the ``--user`` option to ``pip``
    (see :ref:`the pip documentation<https://pip.pypa.io/en/stable/user_guide/#user-installs>`).
```

(has no effect on the rendering)

Reply:
Done.